app/src/main/res/values/strings.xml

```diff
@@ -5,6 +5,11 @@
     <string name="activity_login_title">Log in to Last.fm</string>
     <string name="enable_notification_access">Enable Notification Access</string>
     <string name="enable_notification_access_summary">Required for scrobbling from Apple Music</string>
+    <string name="label_now_playing">Now Playing Track</string>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mataku/SunsetScrob/blob/358f4d3d5d1451906a8d4af2536fd5c14fbe83c0/app/src/main/res/values/strings.xml#L8"></span>The resource <code>R.string.label_now_playing</code> appears to be unused</td>
    </tr>
  </tbody>
</table>